### ISSUE TITLE
Not an issue – method description

### ISSUE THREAD

**Issue by lbinding** [AICONSlab/SynthSegCSVD#1]
Hi all, 

I just wanted to say congratulations for making such an easy-to-use, reliable tool. From some preliminary testing the tool looks great! 

I was hoping to get some more information about how the method so I can start writing the methods of a paper. 

Thank you!! 

All the best,
Lawrence

**Comment by eag** [comment]
Hi Lawrence,  

Thank you very much for the positive feedback!  

A preprint of the WMH segmentation paper will be available, likely later this month, and a link will be added to the Readme page.  

The PVS segmentation method is still under development. We are planning to release an update for this tool later this month with improved sensitivity and no requirement for RORPO filtering.    

In the meantime, please find below references and more information on the current versions.

Best,
Erin

**SynthSegCSVD<sub>WMH</sub>  Reference:** 
Erin Gibson, Joel Ramirez, Lauren Abby Woods, Stephanie Berberian, Julie Ottoy, Christopher J.M. Scott, Fuqiang Gao, Roberto Duarte Coello, Maria Valdes Hernandez, Anthony E. Lang, Carmela M. Tartaglia, Malcolm A. Binns, Robert Bartha, Sean Symons, Richard H. Swartz, Mario Masellis, Alan Moody, Bradley J. MacIntosh, Joanna M. Wardlaw, Sandra E. Black, ONDRI Investigators, ADNI, CAIN Investigators, colleagues from the Foundation Leducq Transatlantic Network of Excellence, Andrew SP Lim, Maged Goubran. (2024). Optimizing WMH Segmentation for diverse clinical datasets with SynthSegCSVD. Poster to be presented at the Human Brain Mapping (HBM) annual meeting, Seoul, South Korea.

****SynthSegCSVD<sub>WMH</sub>  Description:**
SynthSegCSVD<sub>WMH</sub> was developed using a large dataset consisting of over 700 scans sourced from seven multi-site studies, encompassing a range of clinical populations, WMH burdens, and imaging protocols. This extensive dataset was further enriched by the creation of high-fidelity ground truth labels derived from semi-automatically generated and manually edited masks. A novel two-stage segmentation framework was developed that first leverages FreeSurfer's SynthSeg to generate a targeted regional mask, and subsequently combines this mask with the FLAIR image for improved WMH segmentation. Advanced machine learning strategies, including the ensembling of models with distinct precision-recall weightings and test-time augmentation techniques were utilized to ensure robust segmentation performance. The efficacy of SynthSegCSVD<sub>WMH</sub> was rigorously evaluated by benchmarking it against two established, state-of-the-art segmentation tools, HyperMapper and SAMSEG, across various test datasets  

**SynthSegCSVD<sub>PVS</sub>  Reference:**  
E Gibson, J Ramirez, LA Woods, R Sommers, N M Ghahjaverestan, CJM Scott, F Gao, AE Lang, C Marras, DP Breen, MC Tartaglia, MA Binns, R B, S Symons, RH Swartz, M Masellis, SE Black, A Moody, ONDRI Investigators, CAIN Investigators, Colleagues from the Foundation Leducq Transatlantic Network of Excellence, Andrew SP Lim, M Goubran. Examining perivascular spaces (PVS) in cerebral small vessel disease (CSVD) using a novel T1-based automated PVS segmentation tool. (2023). Poster presented at the International Society of Vascular Behavioural and Cognitive Disorder (VASCOG) Conference, Gothenburg, Sweden.

**SynthSegCSVD<sub>PVS</sub> Description:** 
192 T1 and FLAIR images were obtained from various multisite studies (ONDRI, CAHHM, Leducq SVD-PVS, CAIN). Ground truth PVS segmentations were generated using a semi-automated procedure consisting of several steps, including:  1. Segmentation of WMH on FLAIR images using SynthSegCSVD<sub>WMH</sub>;  2. Segmentation of possible PVS objects on denoised T1 images and (multiplicatively) combined denoised T1xFLAIR images using the RORPO filter; 3. Refinement of the RORPO output around WMHs, first by the removal of all possible T1-derived PVS objects connected in 3D to WMHs, followed by the addition/recovery of all possible T1xFLAIR-derived PVS objects within WMHs; 4. Removal of probable non-PVS objects using FreeSurfer GM/CSF ROIs ; and 5. Slice-wise manual editing to correct remaining errors.  Two CNNs were trained using either two or three inputs and the same UNet architecture  (n=141/11/40 training/validation/testing split; 1000 training epochs; Tversky Loss and beta=0.6; Adam optimizer and learning rate=0.00014).
